_migration-assistant/migration-phases/backfill.md

```diff
@@ -82,6 +82,11 @@ Depending on the size of the data in the source cluster and the bandwidth alloca
 
 From the snapshot you created of your source cluster, you can begin backfilling documents into the target cluster. Once you have started this process, a fleet of workers will spin up to read the snapshot and reindex documents into the target cluster. This fleet of workers can be scaled to increased the speed at which documents are reindexed into the target cluster.
 
+### Cloudwatch Metrics and Dashboard
+The Migration Assistant creates a CloudWatch Dashboard you can use to visualize the health and performance of the backfill process.  It brings together the metrics for the backfill workers and, for those migrating to Amazon OpenSearch Service, the target cluster.
+
+You can find the Backfill dashboard in the AWS Console for CloudWatch Dashboards, in the region you have deployed the Migration Assistant.  The metric graphs for your target cluster will be blank until you select the OpenSearch Domain you're migrating to from the drop-down menu at the top of the Dashboard.
```

Comment:
Can we put in the name of the dashboard?

Don't we also have a capture & replay dashboard?

Reply:
(ah, I just realized that this is because this page is about backfill 👍 )